### ISSUE TITLE
Curious question

### ISSUE THREAD

**Issue by Budman17r** [enkur/ATT-Fiber-Check#6]
Did we want to maybe modify the script to report the address.py to a mysql database (which I could in theory host) to allow for better crowdsourcing of the found data.

**Comment by enkur** [comment]
I dont know how to code all that up. If you can host the database and have a way for users to dump the information that would be cool. I am not sure how much bandwidth that would take.

**Comment by Budman17r** [comment]
I dunno either I have an unlimited bandwidth vps.  I'll see what I can do.
 (funny enough I don't know either I'm learning as I go)

**Comment by enkur** [comment]
The person who originally wrote the script eventually migrated to Javascript and also put everything in sql. Here is the link to his project. You can try to figure out from the code to see what he did
https://github.com/mtdoyle/CLSpeedPhantomJS

**Comment by phobozad** [comment]
Letting random people write to a database open to the internet seems like a bad idea.  Also database access over high latency links (aka the internet) has generally bad performance.

A KISS solution might be to create a web form that people can upload their CSV file to to be built into a combined community map/database.  This would in theory allow for moderation/rejection of obviously bad or malicious data.  Would require some minimal amount of time for whoever is running it to perform some rudimentary curating of the data.  Or maybe not...just allow people to dump in random data and hope for the best?

**Comment by joe-eklund** [comment]
Probably the best way would be for some automatic sanitizing. I like the csv file upload idea.

**Comment by enkur** [comment]
Yes the csv upload idea make more sense rather than a script that anyone can run on a database, I agree.

**Comment by Budman17r** [comment]
You make valid points good sir
